### ISSUE TITLE
Training details on VOC dataset (VOC数据集上训练的相关细节) ?

### ISSUE THREAD

**Issue by Hanqer** [speedinghzl/CCNet#42]
So the original paper referred the experimental results on VOC dataset.

However, there is no VOC-related training schemes inside the code.

Could someone kindly provide training details (or code) on the VOC dataset?

> e.g. base learning rate, learning rate decreasing strategy, batchsize

原文中有在 VOC 数据集上的结果，但是代码中没有提供 VOC 上训练的一些超参数细节。

请问能提供一下 VOC 数据集上训练的相关训练代码或者超参数吗？

> 例如 初始学习率，学习率下降方式，batchsize等

**Comment by speedinghzl** [comment]
Hi @Hanqer , we do not carry out experiments on VOC dataset and can not provide training details.
